doc/reference.md

```diff
@@ -301,6 +302,63 @@ import 'package:square_in_app_payments/in_app_payments.dart';
 ```
 --- 
 
+### startBuyerVerificationFlow
+
+Starts buyer verfication flow for card on file(cof). Displays verification view for some geographies. The method takes two callback parameters which correspond
+to the possible results of the request. 
+
+Parameter       | Type                                     | Description
+:-------------- | :--------------------------------------- | :-----------
+onBuyerVerificationSuccess | [BuyerVerificationSuccessCallback](#BuyerVerificationSuccessCallback) | Invoked when card entry with buyer verification is completed successfully.
```

Comment:
This is no longer true. Please reword as:
Invoked when buyer verification on a given payment source id succeeds